examples/lib/defaultHtmlStepUi.js

```diff
@@ -335,8 +341,31 @@ function DefaultHtmlStepUi(_sequencer, options) {
               .data('initValue', step.options[i]);
         }
       }
+      function fallbackCopyTextToClipboard(text) {
+        var textArea = document.createElement("textarea");
+        textArea.value = text;
+        textArea.style.position="fixed";  //avoid scrolling to bottom
+        document.body.appendChild(textArea);
+        textArea.focus();
+        textArea.select();
+      
+        try {
+          var successful = document.execCommand('copy');
+        } catch (err) {
+          console.error('Fallback: Oops, unable to copy', err);
+        }
+      
+        document.body.removeChild(textArea);
+      }
       for (var i in outputs) {
-        if (step[i] !== undefined)
+        if(step.name=='decode-qr'){
```

Comment:
Decode QR isn't the only module that uses this. We want this feature for every module output field.

Reply:
Done